src/cmdstan/print.cpp

```diff
@@ -127,12 +128,10 @@ int main(int argc, const char* argv[]) {
   }
   
   // Prepare header
-  Eigen::Matrix<std::string, Eigen::Dynamic, 1> headers(n);
-  headers << 
-    "Mean", "MCSE", "StdDev",
-    "5%", "50%", "95%", 
-    "N_Eff", "N_Eff/s", "R_hat";
-  
+  std::vector<std::string> headers = {"Mean", "MCSE", "StdDev",
```

Comment:
[optional]
The `=` is optional here; without it, the construct is called directly.

Reply:
Done.